cli/tests/integrations/test_help.py

```diff
@@ -7,6 +7,7 @@ def test_help():
 Usage:
     dcos help --info
     dcos help
+    dcos help <command>
```

Comment:
We need more tests. We need to test success cases and failure cases. For example: `dcos help marathon`, `dcos package install kafka --cli && dcos help kafka`, `dcos help unknown-command`.